sensor_msgs/Image path_marker_img
---
sensor_msgs/Image path_marker_result_img
geometry_msgs/Vector3 path_marker_heading_to
geometry_msgs/Vector3 path_marker_heading_from
geometry_msgs/Point path_marker_center

